src/main/java/noel/tasks/Events.java

```diff
@@ -0,0 +1,48 @@
+package noel.tasks;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Events extends Task {
+
+    final String taskChar = "[E]";
```

Comment:
Similar to `taskChar` in your deadline class, this constant's name should be capitalised